packages/kbn-babel-preset/common_preset.js

```diff
@@ -44,7 +44,11 @@ module.exports = {
         /x-pack[\/\\]plugins[\/\\]infra[\/\\].*[\/\\]graphql/,
         /x-pack[\/\\]plugins[\/\\]siem[\/\\].*[\/\\]graphql/,
       ],
-      plugins: [[require.resolve('babel-plugin-typescript-strip-namespaces')]],
-    },
-  ],
+      plugins: [
```

Comment:
Looks like there's a diff up on line `40`:

feature-secops:
`// strip them off for babel on x-pack infra/siem plugins`

master:
`// strip them off for babel on x-pack infra plugin `

This is probably correct to keep though.

Reply:
It is correct since there will be two projects that need them stripped off so I left the 💬 comment there. Thanks for double checking though.